packages/core/android/google-services.json

```diff
@@ -10,7 +10,7 @@
       "client_info": {
         "mobilesdk_app_id": "1:169668264691:android:3c64b2cb29f7ff3516ed86",
         "android_client_info": {
-          "package_name": "com.example.segmentanalyticsreactnative"
+          "package_name": "com.segmentanalyticsreactnative"
```

Comment:
why is this file in `core`? it contains data particular to your test instance right? we should remove it.